modules/data/generic_pdf.components.json

```diff
@@ -37,5 +37,17 @@
         "selectorData": "download",
         "strategy": "id",
         "groups": []
+    },
+
+    "first-name-field": {
+        "selectorData": "pdfjs_internal_id_638R",
+        "strategy": "id",
+        "groups": []
+    },
+
+    "edited-field": {
+        "selectorData": "[id='pdfjs_internal_id_638R'][value='Mark']",
```

Comment:
I recommend making a change to this selector as suggested for the previous

Reply:
Made the changes. Thanks.